modules/security-context-constraints-rbac.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+[id='role-based-access-to-ssc-{context}']
+= Role-based access to Security Context Constraints
+
+You can specify SCCs as a resource that is handled by RBAC. This allows
+you to scope access to your SCCs to a certain project or to the entire
+cluster. Assigning users, groups, or service accounts directly to an
+SCC retains cluster-wide scope.
+
+To include access to SCCs for your role, specify the `scc` resource
+when creating a role.
+
+----
+oc create role <role-name> --verb=use --resource=scc --resource-name=<scc-name> -n <namespace>
+----
+
+This results in the following role definition:
+
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+...
+  name: role-name <1>
+  namespace: namespace <2> 
+...
+rules:
+- apiGroups:
+  - security.openshift.io <3>
+  resourceNames:
+  - my-permitting-scc <4>
+  resources:
+  - securitycontextconstraints <5>
+  verbs: <6>
+  - use 
+----
+<1> The role's name.
+<2> Namespace of the defined role. Defaults to `default` if not specified.
+<3> The API group that includes the SecurityContextConstraint resource. 
+Automatically defined when `scc` is specified as a resource.
+<4> An example name for an SCC you want to have access.
+<5> Name of the resource group that allows users to specify SCC names in
+the `resourceNames` field.
+<6> A list of verbs to apply to the role. 
+
+A local or cluster role with such a rule allows the subjects that are 
+bound to it with a RoleBinding or a ClusterRoleBinding to use the 
+user-defined SCC called `scc-name`.
+
+[NOTE]
+====
+Because RBAC is designed to prevent escalation, even project administrators 
```

Comment:
It might read fine to someone with better understanding of the subject matter, but I found this sentence hard to read. I think having two because's isn't helping.

Reply:
I've broken this into two sentences. Let me know your thoughts on this section.

Reply:
This has been discussed in IRC - content has been adjusted and approved.